tests/it/queue.rs

```diff
@@ -5,34 +5,32 @@ use broker::queue::{self, Queue, Receiver, Sender};
 async fn racing_senders_err() {
     let _root = set_temp_data_root();
 
-    let _first = Sender::open(Queue::Echo).await.expect("must open first");
-    let err = Sender::open(Queue::Echo)
-        .await
-        .expect_err("must fail to open second");
-    assert_error_stack_snapshot!(&"sender", err);
+    let _first: Sender<Vec<u8>> = Sender::open(Queue::Echo).await.expect("must open first");
+    let second: Result<Sender<Vec<u8>>, _> = Sender::open(Queue::Echo).await;
+    assert_error_stack_snapshot!(&"sender", second.expect_err("must fail to open second"));
 }
 
 #[tokio::test]
 async fn racing_receivers_err() {
     let _root = set_temp_data_root();
 
-    let _first = Receiver::open(Queue::Echo).await.expect("must open first");
-    let err = Receiver::open(Queue::Echo)
-        .await
-        .expect_err("must fail to open second");
-    assert_error_stack_snapshot!(&"receiver", err);
+    let _first: Receiver<Vec<u8>> = Receiver::open(Queue::Echo).await.expect("must open first");
+    let second: Result<Receiver<Vec<u8>>, _> = Receiver::open(Queue::Echo).await;
```

Comment:
~~I'm still seeing test failures here~~

I do see failures, but only when I run `cargo test`. `cargo nextest run` succeeds.

Reply:
Fixing this in the next PR once and for all.